### ISSUE TITLE
DietPi-Software | Emby fails to install (pkg error)

### ISSUE THREAD

**Issue by Fourdee** [MichaIng/DietPi#619]
> http://dietpi.com/phpbb/viewtopic.php?f=11&t=902&p=4019#p4018

```
The following packages have unmet dependencies:
 emby-server : Depends: libembysqlite3-0 but it is not installable
               Depends: embymagick but it is not installable
```

**Comment by Fourdee** [comment]
Same with ```apt-get install emby-server-beta```.
Repo : http://download.opensuse.org/repositories/home:/emby/Debian_8.0/

> https://twitter.com/DietPi_/status/803640254020272129




> http://download.opensuse.org/repositories/home:/emby/Debian_8.0/Packages

```embymagick``` and ```libembysqlite3-0``` are x86_32 x86_64 only. Repo lacks ARMv7+ packages for these new dependencies.

**Comment by Fourdee** [comment]
Options:
- find documentation/method for compiling Emby ourselves: https://github.com/MediaBrowser/Emby.Build/tree/master/builders/emby-server (is this only for Docker?) | https://github.com/MediaBrowser/Emby/tree/master/Tools/Linux_Build_Scripts ?
- find another Emby repo
- wait for OpenSuse Emby repo to support ARMxx

**Comment by Fourdee** [comment]
Lets ask the Emby community:
https://emby.media/community/index.php?/topic/19415-debian/?p=392132

**Comment by Fourdee** [comment]
Solution/workaround: https://emby.media/community/index.php?/topic/19415-debian/#entry392490

**Comment by Fourdee** [comment]
🈯️ ARM64
🈯️ ARMv7+

**Comment by Fourdee** [comment]
🈯️ ARMv7+
🈯️ ARM64

Resolved using 14.04 Emby repo and libjpeg8 from sid.
